src/cursesdef.h

```diff
@@ -1,7 +1,7 @@
 #ifndef CURSESDEF_H
 #define CURSESDEF_H
 
-#if (defined TILES || defined _WIN32 || defined WINDOWS)
+#if (defined TILES || defined SDLTILES || defined _WIN32 || defined WINDOWS)
```

Comment:
Why is this change in this PR?
I'm pretty sure SDLTILES was obsoleted and thus shouldn't be referenced anywhere.